src/pages/steps/CountryPicker.tsx

```diff
@@ -68,7 +73,7 @@ const CountryPicker = ({
 
       <Dropdown.Divider />
 
-      {includeLocationSchool && (
+      {!isCultuurkuurFeatureFlagEnabled && includeLocationSchool && (
```

Comment:
I wonder if it would be better to rename `includeLocationSchool` to `showSchoolLocation` or `showCultuurkuurLocation` ?